controllers/mapController.js

```diff
@@ -110,7 +115,87 @@ const addInvitedUser = async (req, res, next) => {
   }
 };
 
+const inviteNewMember = async (req, res, next) => {
+  const { id } = req.params;
+  const { email } = req.body;
+
+  try {
+    const currentMap = await Map.findById(id).exec();
+    const newUser = await User.findOne({ email }).exec();
+
+    const userId = newUser._id;
+    const verifyInvitedEmail = currentMap.invitationList
+      .map(user => user.email)
+      .some(invitedEmail => invitedEmail === email);
+
+    if (!newUser) {
+      res.json({
+        result: "등록되지 않은 유저입니다.",
+      });
+      return;
+    }
+
+    if (currentMap.members.some(member => member.equals(userId))) {
```

Comment:
여기도 마찬가지로 includes 사용을 조심스레 제안해봅니다..ㅎ_ㅎ

```suggestion
    if (currentMap.members.includes(userId)) {
```

Reply:
더 직관적인 것 같아요~! 좋아요!